tests/functional/adapter/test_ephemeral.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+from dbt.tests.adapter.ephemeral.test_ephemeral import BaseEphemeralMulti
```

Comment:
Why does this not count as a proper import? at same level as all other files in the adapter section of `core`, and i'm importing this current branch i'm testing against in `dev_requirements`

Reply:
It looks like the new test in `dbt-core` is not nested under adapters so that path is not valid.

Reply:
oh so this one is incorrect? https://github.com/dbt-labs/dbt-core/pull/5735/files created a second pr to test this attempt against